### DISCUSSION TITLE
Markdown Magic 3.0.6 no longer uses `markdownMagic` function

### DISCUSSION THREAD

**Discussion by jordanjones243** [AlaskaAirlines/WC-Generator#545] [Announcements]
When upgrading to markdown-magic 3.0.6 in any repo you will see the following error

<img width="938" alt="Screen Shot 2024-05-01 at 11 54 23 AM" src="https://github.com/AlaskaAirlines/WC-Generator/assets/55807669/338cc5b1-7261-4293-9a38-ae5f14b7c611">

**Comment by blackfalcon** [comment]
Yes, I ran into that while trying to update dependencies in a repo as well. We should create an issue with the WC-Generator that addresses this update. 

While we do have a 'template' docs generator in the auro-library project, the document is not being pulled in real-time from that repo into each repo. So we will need an item that will address updating this document in each repo as it is updated to the 3.x version.

**Reply by blackfalcon** [reply]
We also have the same issue with ESLint - https://github.com/orgs/AlaskaAirlines/discussions/541
